pkgs/applications/misc/topydo/default.nix

```diff
@@ -1,29 +1,41 @@
-{ lib, python3Packages, fetchFromGitHub, glibcLocales }:
+{ lib, python3, fetchFromGitHub, fetchpatch, glibcLocales }:
 
-with python3Packages;
-
-buildPythonApplication rec {
+python3.pkgs.buildPythonApplication rec {
   pname = "topydo";
   version = "0.14";
 
   src = fetchFromGitHub {
-    owner = "bram85";
+    owner = "topydo";
     repo = pname;
     rev = version;
     sha256 = "1lpfdai0pf90ffrzgmmkadbd86rb7250i3mglpkc82aj6prjm6yb";
   };
 
-  propagatedBuildInputs = [
+  patches = [
+    # fixes a failing test
+    (fetchpatch {
+      name = "update-a-test-reference-ics-file.patch";
+      url = "https://github.com/topydo/topydo/commit/9373bb4702b512b10f0357df3576c129901e3ac6.patch";
+      hash = "sha256-JpyQfryWSoJDdyzbrESWY+RmRbDw1myvTlsFK7+39iw=";
+    })
+  ];
+
+  nativeCheckInputs = with python3.pkgs; [
+    freezegun
+    mock
+    pylint
```

Comment:
```suggestion
```
we don't want to lint

Reply:
also removed `mock` since that also seems unused